example/src/App.tsx

```diff
@@ -19,12 +19,13 @@ function App() {
       style={{
         display: 'flex',
         padding: '20px',
-        overflow: 'auto',
+          height: '100vh'
```

Comment:
The indentation seems off

Reply:
indeed... somehow my editor doesn't detect the config, and thus doesn't auto lint on save...... incredibly frustrating